### ISSUE TITLE
backend-ethereum:amd64 (= 2.58.1-satoshilabs-1) has no Size information

### ISSUE THREAD

**Issue by 7-of-9** [trezor/blockbook#1049]
`make-all ethereum`
`apt install ./build/backend-ethereum_2.58.1-satoshilabs-1_amd64.deb --reinstall`

Yields:
`W: Repository is broken: backend-ethereum:amd64 (= 2.58.1-satoshilabs-1) has no Size information`

On a fresh pull of this (latest) commit: https://github.com/trezor/blockbook/commit/60fb37da4f0b4f02235ee8f1a43a10c0850f753e

Distro:
Distributor ID: Ubuntu
Description:    Ubuntu 18.04.6 LTS
Release:        18.04
Codename:       bionic

Any pointers, ideas? thx!

**Comment by martinboehm** [comment]
We have not encountered this problem. We install the package using `dpgk -i <package>` but I am not sure if it has anything to do with it.

**Comment by 7-of-9** [comment]
Thanks for the input Martin. The build time seems very fast, like something is not right with it. Trying again with latest commit, will try and post build logs if it doesn't magically resolve :)

**Comment by 7-of-9** [comment]
An error here in build-images, seems it's ignored... not sure if it's relevant...?

`make build-images` => 

docker rmi blockbook-build
Untagged: blockbook-build:latest
Deleted: sha256:a6a563cd2e4daeb68ded826ac88b7434bad0dd8197a200ea1741d6adf1fc35c4
docker rmi blockbook-build-deb
**Error response from daemon: conflict: unable to remove repository reference "blockbook-build-deb" (must force) - container f6558b6b5678 is using its referenced image df6899e010e2
Makefile:81: recipe for target 'clean-deb-image' failed
make: [clean-deb-image] Error 1 (ignored)**
rm -f .bin-image .deb-image  # remove obsolete tag files
make .bin-image .deb-image
make[1]: Entering directory '/home/ubuntu/blockbook2'
Building image blockbook-build from ubuntu:18.04
[+] Building 0.1s (20/20) FINISHED                                                                                                                                        
 => [internal] load .dockerignore                                                                                                                                    0.0s
 => => transferring context: 2B                                                                                                                                      0.0s
 => [internal] load build definition from Dockerfile                                                                                                                 0.0s
 => => transferring dockerfile: 2.18kB                                                                                                                               0.0s
 => [internal] load metadata for docker.io/library/ubuntu:18.04                                                                                                      0.0s
 => [internal] load build context                                                                                                                                    0.0s
 => => transferring context: 30B                                                                                                                                     0.0s
 => [ 1/15] FROM docker.io/library/ubuntu:18.04                                                                                                                      0.0s
 => CACHED [ 2/15] RUN apt-get update &&     apt-get upgrade -y &&     apt-get install -y build-essential git wget pkg-config lxc-dev libzmq3-dev                    0.0s
 => CACHED [ 3/15] RUN mkdir /build                                                                                                                                  0.0s
 => CACHED [ 4/15] RUN if [ -n "" ]; then     echo "Using TCMALLOC";     apt-get install -y google-perftools;     ln -s /usr/lib/libtcmalloc.so.4 /usr/lib/libtcmal  0.0s
 => CACHED [ 5/15] RUN if [ "linux/amd64" = "linux/amd64" ]; then ARCHITECTURE=amd64; elif [ "linux/amd64" = "linux/arm64" ]; then ARCHITECTURE=arm64; elif [ "linu  0.0s
 => CACHED [ 6/15] RUN ln -s /opt/go/bin/go /usr/bin/go                                                                                                              0.0s
 => CACHED [ 7/15] RUN mkdir -p /go                                                                                                                                  0.0s
 => CACHED [ 8/15] RUN echo -n "GO version: " && go version                                                                                                          0.0s
 => CACHED [ 9/15] RUN echo -n "GOPATH: " && echo /go                                                                                                                0.0s
 => CACHED [10/15] RUN cd /opt && git clone -b v7.7.2 --depth 1 https://github.com/facebook/rocksdb.git                                                              0.0s
 => CACHED [11/15] RUN cd /opt/rocksdb && CFLAGS=-fPIC CXXFLAGS=-fPIC PORTABLE=0 make -j 4 release                                                                   0.0s
 => CACHED [12/15] RUN strip /opt/rocksdb/ldb /opt/rocksdb/sst_dump &&     cp /opt/rocksdb/ldb /opt/rocksdb/sst_dump /build                                          0.0s
 => CACHED [13/15] RUN     cleanup() { rm -rf /go/src/github.com/trezor ; } &&     trap cleanup EXIT &&     mkdir -p /go/src/github.com/trezor &&     cd /go/src/gi  0.0s
 => CACHED [14/15] ADD Makefile /build/Makefile                                                                                                                      0.0s
 => CACHED [15/15] WORKDIR /build                                                                                                                                    0.0s
 => exporting to image                                                                                                                                               0.0s
 => => exporting layers                                                                                                                                              0.0s
 => => writing image sha256:a6a563cd2e4daeb68ded826ac88b7434bad0dd8197a200ea1741d6adf1fc35c4                                                                         0.0s
 => => naming to docker.io/library/blockbook-build:latest                                                                                                            0.0s
Image blockbook-build-deb is up to date
make[1]: Leaving directory '/home/ubuntu/blockbook2'

**Comment by martinboehm** [comment]
I do not think we have done some changes to the build process recently. I would clear the docker images and try again.
